packages/create-amplify/src/package-manager-controller/npm_package_manager_controller.ts

```diff
@@ -0,0 +1,50 @@
+import { execa as _execa } from 'execa';
+import { executeWithDebugLogger } from '../execute_with_logger.js';
+import {
+  DependencyType,
+  PackageManagerController,
+} from './package_manager_controller_factory.js';
+
+/**
+ *
+ */
+export class NpmPackageManagerController implements PackageManagerController {
+  protected readonly execa = _execa;
+  protected readonly packageManagerProps = {
```

Comment:
it looks like only a few of these values are actually used in this class. We should only define the ones we care about here. Also, they can just be direct properties of the class, no need to nest under `packageManagerProps`

Reply:
done